chaos_genius/alerts/email_templates/digest_template.html

```diff
@@ -53,13 +53,13 @@ <h3>Top 10 Anomalies</h3>
         <span style="color: lightgrey;">
             changed to <strong>{{point.y}}</strong>
         {% if point.percent_change is string and "inf" not in point.percent_change %}
-        (<span>{{ point["change_message"] }}</span>)
+        (<span>{{ point.formatted_change_percent }}</span>)
```

Comment:
some concisness possible for whole logic of if else , i also feel this might decrease readability although, what do you think

```suggestion
        {% if point.percent_change is string and "inf" not in point.percent_change %}
        (<span>
        {% elif "+inf" in str(point.percent_change) or point.percent_change > 0 %}
        (<span style="color: lightgreen;">
        {% elif "-inf" in str(point.percent_change) or point.percent_change < 0 %}
        (<span style="color: red;">
        {% else %}
        (<span>
        {% endif %}
        {{ point.formatted_change_percent }}</span>)
```